### ISSUE TITLE
Referential Actions as preview feature

### ISSUE THREAD

**Issue by moacyrsouza** [prisma/prisma#11297]
### Bug description

I have some relations with Referencial Actions set for onDelete and onUpdate in my schema. When I run the application, I get an error saying I can't use the Referencial Actions without specifying it as a previewFeature. If I it my application works fine but I get a `Preview feature "referentialActions" is deprecated. The functionality can be used without specifying it as a preview feature.` warning. Also according to Prisma documentation, it is a released feature and I should not be required to specify it as preview.

### How to reproduce

1. Make schema.prisma with some relations using onDelete and onUpdate without specifying "referentialActions" as preview feature
2. Create migration and apply it
3. Try to create a record for a referencing/referenced mode

### Expected behavior

After applying the migrations without "referentialActions" specified as a preview feature, the application should work

### Prisma information

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

generator client {
  provider = "prisma-client-js"
  previewFeatures = ["referentialActions"] // I get the warning here
}

I can't share more than this

### Environment & setup

- OS: Alpine Linux (from node:14.17-alpine Docker image)
- Database: PostgreSQL 13.1
- Node.js version: 14.17

### Prisma Version

```
3.8.1
```

**Comment by janpio** [comment]
> When I run the application, I get an error saying I can't use the Referencial Actions without specifying it as a previewFeature.

Can you share the full message you are getting?
Are you sure you are running 3.8.1 - that should not be happening with that version at all.

**Comment by moacyrsouza** [comment]
Sorry for taking so long, but I just tried to run a create operation and it worked. I needed to deploy my app, so I kept the "referentialActions" as preview feature in my schema, even though I was getting warnings. Now I removed it and created a new migration. It is working perfectly.

But I was able to retrieve a snippet of the error message I was getting: "[name of the referencing model] must not specify the `onDelete` argument in the @relation attribute without enabling the `referentialActions` preview feature"
